tests/Umbraco.Tests.UnitTests/Umbraco.Core/PropertyEditors/DataValueReferenceFactoryCollectionTests.cs

```diff
@@ -173,6 +173,44 @@ public void GetAllReferences_Invariant_With_IDataValueReference_Editor()
         Assert.AreEqual(trackedUdi4, result.ElementAt(1).Udi.ToString());
     }
 
+    [Test]
+    public void GetAutomaticRelationTypesAliases_ContainsDefault()
+    {
+        var collection = new DataValueReferenceFactoryCollection(Enumerable.Empty<IDataValueReferenceFactory>);
+        var propertyEditors = new PropertyEditorCollection(new DataEditorCollection(Enumerable.Empty<IDataEditor>));
+        var properties = new PropertyCollection();
+
+        var resultA = collection.GetAutomaticRelationTypesAliases(propertyEditors).ToArray();
+        var resultB = collection.GetAutomaticRelationTypesAliases(properties, propertyEditors).ToArray();
+
+        Assert.AreEqual(resultA.Length, Constants.Conventions.RelationTypes.AutomaticRelationTypes.Length);
+        Assert.AreEqual(resultA.Length, Constants.Conventions.RelationTypes.AutomaticRelationTypes.Length);
+
+        foreach (var alias in Constants.Conventions.RelationTypes.AutomaticRelationTypes)
+        {
+            Assert.Contains(alias, resultA);
+            Assert.Contains(alias, resultB);
+        }
```

Comment:
Should be wrapped in Assert.Multiple to ensure we get more data if it fails

Reply:
I've wrapped the assertions and added messages to easily see which assertion failed 👍🏻